java/refactoring.java/nbproject/project.xml

```diff
@@ -52,6 +52,15 @@
                         <specification-version>1.0</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.api.lsp</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <release-version>1</release-version>
+                        <specification-version>1.8</specification-version>
```

Comment:
The new version of LSP API is 1.9. Version 1.8 introduced StructureElement, which is changed in this new version 1.9.